scene/gui/graph_node.cpp

```diff
@@ -336,7 +336,8 @@ void GraphNode::_notification(int p_what) {
 
 			int width = get_size().width - sb_panel->get_minimum_size().x;
 
-			if (get_child_count() > 0) {
+			// Take into account the HboxContainer child
+			if (get_child_count() > 1) {
```

Comment:
Would be better to use the boolean argument to exclude internal nodes: https://github.com/godotengine/godot/blob/79de2eaae7fb3bb8f13031ac2ec9b19128b6f471/scene/main/node.h#L431
Then the comment is likely not needed as it's bound to become outdated in the future if internal children change.